.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
+git add .
```

Comment:
Great job ivad. for pre-commit I am able to amend with a console.log statement, so recheck again if it is working as expected.

Reply:
is that issue only for `git commit --amend`? for just  `git commit` Eslint is able to detect the console?

Reply:
On Node environment console statements are allowed as they can be used apart from debugging but also as a way to present information to the admin. But you can add a new Eslint rule to get them as warning

Reply:
We are using Winston for logging so that will not be an issue
the Eslint rule will be a warning but committing will be blocked

Reply:
The idea is we do not want consoles in production, @Yvad60 update the lint script  and pre-commit hook to stop us from making any commits when we have errors or warnings